packages/tree/src/notebook-tree.ts

```diff
@@ -4,6 +4,14 @@ import { TabPanel } from '@lumino/widgets';
 
 import { INotebookTree } from './token';
 
+/**
+ * The namespace for command IDs.
+ */
+export namespace CommandIDs {
```

Comment:
With https://github.com/jupyter/notebook/pull/6588/files#r1067176298 above, these `CommandIDs` could be moved to the `tree-extension` package.